lib/createDIContainer.js

```diff
@@ -328,13 +339,17 @@ async function createDIContainer(options) {
         isolatedJsonSchemaValidatorSnapshot,
         isolatedSTUnserializationOptions,
         transactionalStateRepository,
+        additionalDppOptions,
       );
 
       return unserializeStateTransitionFactory(createIsolatedDpp);
     }).singleton(),
 
     transactionalDpp: asFunction((transactionalStateRepository) => (
-      new DashPlatformProtocol({ stateRepository: transactionalStateRepository })
+      new DashPlatformProtocol({
+        stateRepository: transactionalStateRepository,
+        ...additionalDppOptions,
```

Comment:
I would apply `dppOptions` and then `stateRepository` in order to avoid future bugs.